docs/translation.md

```diff
@@ -79,6 +81,21 @@ For example:
 `zulipLocalizations.subscribedToNChannels(store.subscriptions.length)`.
 
 
+## Adding a new language
+
+ARB files for new languages are automatically created in Pull Requests generated
+[the update-translations GitHub workflow](/.github/workflows/update-translations.yml).
+However, this won't make them appear in settings.
+
+When a language has a good percentage of strings translated, we should add it to
```

Comment:
Does Weblate show the percentage? Let's try to match what zulip-mobile uses for a threshold; I think it might be 5%?